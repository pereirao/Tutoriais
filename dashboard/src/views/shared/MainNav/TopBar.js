// @flow

import React from 'react';
import AccountLink from './AccountLink';
import StoreTitle from './StoreTitle';
import AppTitle from './AppTitle';
import style_sheet from './MainNav.module.css';

type TopBarProps = { current_path: string };
const TopBar = ({current_path}: TopBarProps) => (
  <div className={style_sheet.topBarWrapper}>
    <AppTitle current_path={current_path}/>
    <StoreTitle/>
    <AccountLink/>
  </div>
);


export default TopBar;
