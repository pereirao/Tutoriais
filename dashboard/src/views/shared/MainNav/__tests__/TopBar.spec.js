import React from 'react';

import TopBar from '../TopBar';

it('renders TopBar correctly', () => {
  expect(shallow(<TopBar />)).toMatchSnapshot();
});
