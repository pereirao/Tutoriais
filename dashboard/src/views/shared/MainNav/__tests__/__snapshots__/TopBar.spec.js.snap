// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TopBar correctly 1`] = `
<div
  className="topBarWrapper"
>
  <AppTitle />
  <Connect(StoreTitle) />
  <Connect(AccountLink) />
</div>
`;
